cmd/lakectl/cmd/branch.go

```diff
@@ -205,14 +205,14 @@ var branchShowCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		client := getClient()
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```

Comment:
```suggestion
		fmt.Println("Branch: %s", u)
```

Reply:
kept Printf and dropped the call to String()